photon-core/src/main/java/org/photonvision/common/networking/NetworkManager.java

```diff
@@ -276,9 +282,14 @@ private Runnable deviceStatus(String devName) {
                     }
                 }
                 last.carrier = carrier;
-            } catch (Exception e) {
-                logger.error("Could not check network status", e);
-            }
-        };
+                } catch (Exception e) {
+                    logger.error("Could not check network status for " + devName, e);
+                    // Cancel the task to avoid spamming the logs
+                    logger.info("Cancelling task " + taskName);
+                    TimedTaskManager.getInstance().cancelTask(taskName);
```

Comment:
Just reviewing the diff - if this task is canceled, do we have a way to bring it back if the interface comes back up?

Reply:
The cancellation is only triggered when the this throws an error, which I think only happens if `Files.readstring()` fails. An alternative is to keep the task but not keep logging the error so that if it stops throwing the monitor will continue working. I'll push a commit with this logic.

Reply:
Which I think could only happen if the file doesn't exist? Maybe we need something like ROS_LOG_ONCE lol.

Reply:
That's my assumption. But maybe there could be another filesystem problem that could cause it to transiently fail. The only downside of not cancelling is that it leaves the monitor running and triggering an exception every 5 seconds.